# Fix timing of the GenericAgent, so that syslog is not spammed down.
# Closes: #639504

diff -Naur otrs2-3.0.10+dfsg1.orig/var/cron/generic_agent.dist otrs2-3.0.10+dfsg1/var/cron/generic_agent.dist
--- otrs2-3.0.10+dfsg1.orig/var/cron/generic_agent.dist	2011-08-31 20:47:23.000000000 +0200
+++ otrs2-3.0.10+dfsg1/var/cron/generic_agent.dist	2011-08-31 20:48:23.532855848 +0200
@@ -10,7 +10,7 @@
 # --
 
 # start generic agent every 20 minutes
-*/20 * * * * otrs test -x $HOME/bin/otrs.GenericAgent.pl && $HOME/bin/otrs.GenericAgent.pl > /dev/null
+19,39,59 * * * * otrs test -x $HOME/bin/otrs.GenericAgent.pl && $HOME/bin/otrs.GenericAgent.pl > /dev/null
 
 # example to execute otrs.GenericAgent.pl on 23:00 with
 # Kernel::Config::GenericAgentMove job file
